src/land/ascat_ssm2ioda.py

```diff
@@ -65,19 +68,20 @@ def _read(self):
             self.varAttrs[iodavar, iconv.OvalName()]['coordinates'] = 'longitude latitude'
             self.varAttrs[iodavar, iconv.OerrName()]['coordinates'] = 'longitude latitude'
             self.varAttrs[iodavar, iconv.OqcName()]['coordinates'] = 'longitude latitude'
-            self.varAttrs[iodavar, iconv.OvalName()]['units'] = 'percent'
-            self.varAttrs[iodavar, iconv.OerrName()]['units'] = 'percent'
-            self.varAttrs[iodavar, iconv.OqcName()]['units'] = 'unitless'
+            self.varAttrs[iodavar, iconv.OvalName()]['units'] = '1'
+            self.varAttrs[iodavar, iconv.OerrName()]['units'] = '1'
 
         # open input file name
         ncd = nc.Dataset(self.filename, 'r')
         # set and get global attributes
-        AttrData["satellite"] = ncd.getncattr('source')
+        # A satellite ID is expected for satellite attribute, use sensor instead
+        AttrData["sensor"] = ncd.getncattr('source')
         AttrData['platform'] = ncd.getncattr('platform_long_name')
```

Comment:
Shouldn't be `instrumentIdentifier` and `satelliteIdentifier` instead of `sensor` and `platform`?

Reply:
Yes, thanks. Fixed.

Reply:
This turned out to be an incorrect change, because sensor and platform are being used here as global attributes whereas the two suggested by Fabio are intended for MetaData in the case when different Locations would potentially have different satellite ID (example being SatWinds).  So I reverted these back again.

Reply:
Oh, great! Thanks for checking, @gthompsnJCSDA! That was my confusion.